js-exercises/union/union.js

```diff
@@ -0,0 +1,103 @@
+Object.compare = (obj1, obj2, iterateObj1 = true) => {
```

Comment:
Yes, `obj2` is also iterable, the variable `iterateObj1` is used to whether to iterate on `obj1` or not because `obj2` may have more entries then `obj1` to check this I am skipping iteration on `obj1` while iterating `obj2` on line 27. I will check for better logic and implementation.